### PR TITLE
Allow modules named "V1" etc

### PR DESCRIPTION
**Pull request by brandonc**

### CODE REVIEW

File: config.reek [hunk 1]
```diff
@@ -35,3 +35,8 @@ LongParameterList:
 # This one just makes sure the Class/Module has a comment. Dumb.
 IrresponsibleModule:
   enabled: false
+
+# Allow modules named "V2", "V1", etc
+UncommunicativeModuleName:
+  accept:
+    - !ruby/regexp /V[0-9]$/
```
Comment (paul) [inline config.reek:60]: Should the match only modules named `V1`, or are things like `FooV1` acceptable? If the former, do we need to anchor the regex?

Comment (brandonc) [reply config.reek:60]: it compares the entire module name, example: "Napa::V1" so it could possibly be modified to `/^|(::)V[0-9]$/`

**Comment by paul** [comment]
Do you mind updating the commit message explaining "why" we should allow those module names? I'd like it if this repo had lots of documentation about why we chose these rules going forward.

**Comment by brandonc** [comment]
@paul revised

**Comment by paul** [comment]
Thanks!

**Pull request merged**